core/src/events.rs

```diff
@@ -477,18 +477,18 @@ impl KeyCode {
     pub const MOUSE_MIDDLE: KeyCode = KeyCode(4);
     pub const BACKSPACE: KeyCode = KeyCode(8);
     pub const TAB: KeyCode = KeyCode(9);
-    pub const RETURN: KeyCode = KeyCode(13);
+    pub const ENTER: KeyCode = KeyCode(13);
```

Comment:
Can you add a reference to https://docs.ruffle.rs/en_US/FlashPlatform/reference/actionscript/3/flash/ui/Keyboard.html#summaryTableConstant in the coment on line 469?

Reply:
Done!